clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleHttpLoadBalancerCachingAgent.groovy

```diff
@@ -211,13 +214,56 @@ class GoogleHttpLoadBalancerCachingAgent extends AbstractGoogleCachingAgent impl
               httpHealthChecksRequest: httpHealthChecksRequest,
               groupHealthRequest: groupHealthRequest,
           )
+          def targetHttpsProxyCallback = new TargetHttpsProxyCallback(
+              googleLoadBalancer: newLoadBalancer,
+              urlMapRequest: urlMapRequest,
+              backendServiceRequest: backendServiceRequest,
+              httpHealthChecksRequest: httpHealthChecksRequest,
+              groupHealthRequest: groupHealthRequest,
+          )
 
+          // We make two requests here with the same target proxy name -- this will cause problems if there is
+          // a Http and Https target proxy with the same name. This shouldn't happen in practice, since Spinnaker
+          // will control the load balancer namespace.
+          // The intent here is that exactly one of the following calls will succeed and one will fail, and
+          // the rest of the LB resource graph is built from there.
           compute.targetHttpProxies().get(project, targetProxyName).queue(targetProxyRequest, targetProxyCallback)
+          compute.targetHttpsProxies().get(project, targetProxyName).queue(targetProxyRequest, targetHttpsProxyCallback)
         }
       }
     }
   }
 
+  // Note: The TargetProxyCallbacks assume that each proxy points to a unique urlMap.
+  class TargetHttpsProxyCallback<TargetHttpsProxy> extends JsonBatchCallback<TargetHttpsProxy> implements FailureLogger {
+    GoogleHttpLoadBalancer googleLoadBalancer
+    BatchRequest urlMapRequest
+
+    // Pass through objects
+    BatchRequest backendServiceRequest
+    BatchRequest httpHealthChecksRequest
```

Comment:
Seems like this might be better named `httpHealthCheckRequest`, instead of `httpHealthChecksRequest`.